python/lsst/ap/association/packageAlerts.py

```diff
@@ -0,0 +1,247 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ("PackageAlertsConfig", "PackageAlertsTask")
+
+import os
+import tempfile
+
+import lsst.alert.packet as alertPack
+import lsst.geom as geom
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+from lsst.utils import getPackageDir
+
+
+"""Methods for packaging Apdb and Pipelines data into Avro alerts.
+"""
+
+
+class PackageAlertsConfig(pexConfig.Config):
+    """Config class for AssociationTask.
+    """
+    schemaFile = pexConfig.Field(
+        dtype=str,
+        doc="Location to write alerts to.",
+        default=os.path.join(getPackageDir("sample_avro_alert"),
+                             "schema/2/1/lsst.alert.avsc"),
+    )
+    cutoutSize = pexConfig.RangeField(
+        dtype=int,
+        min=0,
+        max=1000,
+        default=30,
```

Comment:
I note for the future that the DPDD says the cutout should be the larger of the detected footprint size or 30 pixels.

Reply:
I'll check and see if I have access to the footprint in the catalog but not sure if I do. Also, considering the cutout between the two different images and possible difference in pixel scale between them, the choice of a fixed max size or footprint based size isn't that clear.

Reply:
Added a ticket to do this: DM-24926